test/unitTests.js

```diff
@@ -38,7 +58,7 @@ describe('Incorrect email and correct password log in test', function() {
           assert.equal(response,null);
            })
        });
-
+describe('Incorrect email and incorrect password log in test', function() {
```

Comment:
This is a duplicate. It must have got messed up while we were fixing conflicts for a different merge. I will remove it in my next PR because I already need to make a few changes to this.